lib/apple.js

```diff
@@ -11,12 +11,53 @@ var errorMap = {
 	21008: 'This receipt is a production receipt, but it was sent to the sandbox service for verification.'
 };
 
+var config = null;
 var sandboxHost = 'sandbox.itunes.apple.com';
 var liveHost = 'buy.itunes.apple.com';
 var path = '/verifyReceipt';
 
+function isValidConfigKey(key) {
+	return key.match(/^apple/);
+}
+
+module.exports.readConfig = function (configIn) {
+	if (!configIn) {
+		// no apple iap or password not required
+		return;
+	}
+	config = {};
+	var config_value_set = false;
```

Comment:
Oops sorry, I missed this one last time..
Would you change  the variable name here as well?